src/app/stargazers/stargazers.component.html

```diff
@@ -40,7 +40,7 @@ <h5>{{ stargazer.username }}</h5>
             </a>
             <h6 class="text-break" *ngIf="stargazer.orcid">
               ORCID <img src="../../assets/images/account-logos/orcid.svg" alt="ORCID iD logo" class="orcid-id-logo site-icons-small" />
-              <a href="https://orcid.org/{{ stargazer.orcid }}" style="font-size: 0.8em">{{ stargazer.orcid }}</a>
+              <a href="https://orcid.org/{{ stargazer.orcid }}" style="font-size: 0.9em">{{ stargazer.orcid }}</a>
```

Comment:
It was like this, but per the [Frontend Guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide), we should avoid inline styles. Also the guideline recommends using `rem`s, although that's more as opposed to using pixels instead of `em`s.

Reply:
Poked around the places we have ORCID IDs and it seems like this is the only place where we define a specific font-size. It might be better to remove the font size style entirely if the ORCID ID line doesn't overflow to the next line